### DISCUSSION TITLE
✅ Allow to style Links in Site Settings

### DISCUSSION THREAD

**Discussion by theavic** [elementor/elementor#22730] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

It would be a nice to have to be able to style links in site settings

### Describe the solution you'd like

I could select some standard and fancy link styles in site global settings

### Describe alternatives you've considered

Manually setting styles

**Comment by nicholaszein** [comment]
Hi @theavic!

Thank you for contributing with your suggestion! 🙏 

This feature is already available in the Elementor **`Editor`** > **`Site Settings`** > **`Theme Style`** > **`Typography`** > **`Link`** section since version **3.0.0**, which was released on August 2020.

You can learn more about it in this Help Center article:
https://elementor.com/help/theme-style-global-settings/

I'm closing this Discussion as Solved!

Kind regards
